### PR TITLE
Add Background Sample

### PR DESCRIPTION
**Pull request by waprin**

### CODE REVIEW

File: appengine/background/app.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+application: cloud-samples-tests
```
Comment (jonparrott) [inline appengine/background/app.yaml]: No application or version sections in app.yaml, please.

File: appengine/background/main.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+"""
+Sample application that demonstrates how to use the App Engine background
+threads.
+
+"""
+
+# [START all]
+import logging
+import webapp2
```
Comment (jonparrott) [inline appengine/background/main.py]: webapp2 is a third-party import, it should go after `from google.appengine`.

File: appengine/background/main_test.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+def app(cloud_config, testbed):
+    main.PROJECTID = cloud_config.project
+    return webtest.TestApp(main.app)
+
+
+def test_anonymous(app):
+    app.get('/change')
+    response = app.get('/')
+    assert response == 'Dog'
+    time.sleep(5)
```
Comment (jonparrott) [inline appengine/background/main_test.py]: just mock time.sleep() so you don't have to do this.

**Comment by waprin** [comment]
@jonparrott take another look...tests are kind of a joke now but alternative is to implement system thread

File: appengine/background/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Using Background Threads from Google App Engine
+
+This example shows how to use manual or basic scaling to start App Engine background threads.
+
+See the [documentation on modules](https://cloud.google.com/appengine/docs/python/modules/) for
+more information.
+
+
+## Setup
```
Comment (jonparrott) [inline appengine/background/README.md]: This can just be removed.

File: appengine/background/main.py [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Sample application that demonstrates how to use the App Engine background
+threads.
+
+app.yaml scaling must
```
Comment (jonparrott) [inline appengine/background/main.py]: Just remove this?

**Comment by jonparrott** [comment]
LGTM after documentation nits.

File: appengine/background/README.md (hunk 4 above)
Comment (waprin) [inline appengine/background/README.md]: fixed

File: appengine/background/main.py (hunk 5 above)
Comment (waprin) [inline appengine/background/main.py]: sorry for sentence fragment, seems useful to mention this won't work with default app.yaml

**Pull request merged**